frontend/src/pages/groupSettings/GroupSettings.tsx

```diff
@@ -77,70 +64,34 @@ const GroupSettings: React.FC = () => {
-                    </HintBody>
-                  </Hint>
-                </>
-              )}
-              {groupSettings.errorAdmin && (
-                <Alert
-                  isInline
-                  variant="warning"
-                  title={`Group no longer exists`}
-                  actionClose={
-                    <AlertActionCloseButton
-                      onClose={() => {
-                        setGroupSettings({ ...groupSettings, errorAdmin: undefined });
-                      }}
-                    />
-                  }
-                >
-                  <p>{groupSettings.errorAdmin}</p>
-                </Alert>
-              )}
-            </FormGroup>
+            <FormGroupSettings
+              title="Data Science administrator groups"
+              body="Select the OpenShift groups that contain all Data Science administrators."
+              footer="View, edit, or create groups in OpenShift under User Management"
+              fieldId="admin-groups"
+              groupsField={GroupsConfigField.ADMIN}
+              items={groupSettings.adminGroups}
+              error={groupSettings.errorAdmin}
+              hintText="All cluster admins are automatically assigned as Data Science administrators."
+              handleMenuItemSelection={handleMenuItemSelection}
+              handleClose={() => {
+                setGroupSettings({ ...groupSettings, errorUser: undefined });
+              }}
+            />
 
-            <FormGroup fieldId="user-groups" label={`Data Science user groups`}>
-              <Text>{`Select the OpenShift groups that contain all Data Science users.`}</Text>
-              <MenuOptionMultiSelect
-                initialState={groupSettings.allowedGroups}
-                onChange={(newState) => handleMenuItemSelection(newState, GroupsConfigField.USER)}
-              />
-              {!groupSettings.errorUser && (
-                <HelperText>
-                  <HelperTextItem variant="indeterminate">{CARD_FOOTER}</HelperTextItem>
-                </HelperText>
-              )}
-              {groupSettings.errorUser && (
-                <Alert
-                  isInline
-                  variant="warning"
-                  title={`Group no longer exists`}
-                  actionClose={
-                    <AlertActionCloseButton
-                      onClose={() => {
-                        setGroupSettings({ ...groupSettings, errorUser: undefined });
-                      }}
-                    />
-                  }
-                >
-                  <p>{groupSettings.errorUser}</p>
-                </Alert>
-              )}
-            </FormGroup>
+            <FormGroupSettings
+              title="Data Science user groups"
+              body="Select the OpenShift groups that contain all Data Science users."
+              footer="View, edit, or create groups in OpenShift under User Management"
```

Comment:
Would `footer` not just be part of this component, why benefit do we get passing it in?